CONTRIBUTING.md

```diff
@@ -109,6 +109,85 @@ See also:
 
 - [Developing](#developing)
 
+# Working with Unreleased Forc and Fuel-Core Functionality
+
+When you need to work with unreleased versions of `forc` or `fuel-core`, you can specify a git branch instead of a version number.
```

Comment:
```suggestion
When you need to work with unreleased versions of `forc` or `fuel-core`, you can specify a git branch instead of a version number in the respective `VERSION` file.
```